projects/plugins/boost/tests/e2e/specs/critical-css/critical-css.test.js

```diff
@@ -88,29 +87,38 @@ test.describe( 'Critical CSS module', () => {
 			'#jetpack-boost-notice-critical-css-regenerate a[href*="jetpack-boost"]'
 		);
 		const jetpackBoostPage = await JetpackBoostPage.init( page );
-		expect(
-			await jetpackBoostPage.waitForCriticalCssGenerationProgressUIVisibility(),
-			'Critical CSS generation progress indicator should be visible'
-		).toBeTruthy();
+
```

Comment:
I definitely like waiting for the meta UI instead of both the progress bar and the meta UI to be visible.